### PR TITLE
Add adversarial demucs

### PR DESCRIPTION
**Pull request by mosem**

### CODE REVIEW

File: denoiser/batch_solvers/generator_bs.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
 import torch
-import logging
 
 import torch.nn.functional as F
 
 from denoiser.batch_solvers.batch_solver import BatchSolver
 
 from denoiser.stft_loss import MultiResolutionSTFTLoss
 
-logger = logging.getLogger(__name__)
```
Comment (Or-Tal) [inline denoiser/batch_solvers/generator_bs.py:10]: don't we need this? erasing this explicitly assumes global logger variable

File: denoiser/models/modules.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import math
```
Comment (Or-Tal) [inline denoiser/models/modules.py:1]: see previous pull requests comment regarding this being a very long file

File: launch_valentini.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 python train.py \
```
Comment (Or-Tal) [inline launch_valentini.sh:9]: please have a separate .sh file for testing keep this example file with demucs

**Review by Or-Tal — CHANGES_REQUESTED** [review]
Hi please see my comments , I don't like the adversarial flag it adds a non necessary flag to all of our config files instead of being configuration specific.

File: denoiser/batch_solvers/generator_bs.py (hunk 1 above)
Comment (mosem) [reply denoiser/batch_solvers/generator_bs.py:10]: This was only for debugging purposes.

**Pull request merged**